components/ExpensesOutput/ExpenseItem.js

```diff
@@ -0,0 +1,66 @@
+import { Pressable, Text, View, StyleSheet } from "react-native";
+import { GlobalStyles } from "../../constants/styles";
+import { getFormattedDate } from "../../util/date";
+
+function ExpenseItem({ description, amount, date }) {
+    function expensePressHandler() {}
```

Comment:
Needs to be wrapped with useCallback() hook. Also try using arrow functions for declaring functions.

Reply:
Ok